lib/resources/CreditNotes.js

```diff
@@ -5,11 +5,12 @@ const stripeMethod = StripeResource.method;
 
 module.exports = StripeResource.extend({
   path: 'credit_notes',
+
   includeBasic: ['create', 'list', 'retrieve', 'update'],
 
   voidCreditNote: stripeMethod({
     method: 'POST',
-    path: '{id}/void',
+    path: '/{id}/void',
```

Comment:
Is that correct? Other paths do not have a leading `/`.

Reply:
@ob-stripe Some do, some don’t. I’m assuming they should all be consistent at the very least and unless I’m misremembering, @rattrayalex-stripe suggested leading `/`

Reply:
I did. I'm pretty sure we have something that normalizes this. just in case, @irace-stripe can you check that there is test coverage for credit notes?

Reply:
There seems to be, though I’m not 100% sure if these tests cover the difference between leading `/` and no leading `/`? https://github.com/stripe/stripe-node/blob/master/test/resources/CreditNotes.spec.js

Reply:
https://github.com/stripe/stripe-node/blob/master/test/resources/CreditNotes.spec.js#L62-L65 seems to give us coverage – if the leading slash was problematic, it would fail since the request would include a `//`.